man/docker-run.1.md

```diff
@@ -52,6 +52,7 @@ docker-run - Run a command in a new container
 [**--restart**[=*RESTART*]]
 [**--rm**[=*false*]]
 [**--security-opt**[=*[]*]]
+[**-s**|**--stop-signal**[=*SIGNAL*]]
```

Comment:
I actually prefer `stop-signal` too, that was the first implementation and that's why the doc still mentions it. For other conversations it seemed better to call it kill-signal, but I'm open to change it back.

/cc @crosbymichael